### PR TITLE
Changes wanReplicationRef config to support republishing

### PR DESCRIPTION
**Pull request by emrahkocaman**

### CODE REVIEW

**Comment by mesutcelik** [comment]
test failure

```
8:45:48 Tests in error: 
18:45:48   XMLConfigBuilderTest.testConfig2Xml2FullConfig:266->testConfig2Xml2Config:275 » InvalidConfiguration
```

**Comment by emrahkocaman** [comment]
Typo fixed, thanks @mesutcelik

File: hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd [hunk 1]
```diff
@@ -1977,5 +1977,6 @@
     <xs:complexType name="wan-replication-ref">
         <xs:attribute name="name" use="required" type="xs:string"/>
         <xs:attribute name="merge-policy" use="required" type="xs:string"/>
+        <xs:attribute name="republish-enabled" use="optional" type="xs:boolean" default="false"/>
```
Comment (mdogan) [inline hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd]: `republishing-enabled` sounds more correct to me... even `enable-republishing`...

Comment (emrahkocaman) [inline hazelcast-spring/src/main/resources/hazelcast-spring-3.5.xsd]: Changed to `republishing-enabled`

**Comment by mesutcelik** [comment]
verify

**Pull request merged**